cupy/testing/helper.py

```diff
@@ -1253,3 +1254,19 @@ class NumpyAliasValuesTestBase(NumpyAliasTestBase):
 
     def test_values(self):
         assert self.cupy_func(*self.args) == self.numpy_func(*self.args)
+
+
+class CUBMockTest:
```

Comment:
This can be useful for other libraries such as cutensor, maybe we can name it with something like
`AssertFunctionIsCalled`?

Reply:
```suggestion
class AssertFunctionIsCalled:
```